low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -224,3 +224,5 @@ SYSTEM_INFORMATION:
    sspl_log_file_path: /var/log/cortx/sspl/sspl.log
    log_level: INFO
    environment: PROD
+   syslog_host: localhost
```

Comment:
please remove syslog conf, no remote logging enabled in Cortxv2, so we this is not needed. Code can hardcode to this host & port

Reply:
Removed. Now we refer the hardcode value from sspl_constants.py
Done.